openadapt/crud.py

```diff
@@ -109,28 +116,133 @@ def insert_recording(recording_data):
 
 
 def get_latest_recording():
-    return (
-        db
-        .query(Recording)
-        .order_by(sa.desc(Recording.timestamp))
-        .limit(1)
-        .first()
+    return db.query(Recording).order_by(sa.desc(Recording.timestamp)).limit(1).first()
+
+
+def get_recording_by_id(recording_id):
+    return db.query(Recording).filter_by(id=recording_id).first()
+
+
+def export_sql(recording_id):
```

Comment:
@Mustaballer can you please move this and the below changes in this file to `db.py`?

Reply:
addressed in https://github.com/MLDSAI/OpenAdapt/pull/223/commits/dafca05e7547c0ba835a3e70f74ed87bc7520a70